src/ConnectedMode/ConnectedModePackage.cs

```diff
@@ -67,6 +69,9 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             issueServerEventsListener = componentModel.GetService<IIssueServerEventsListener>();
             issueServerEventsListener.ListenAsync().Forget();
 
+            qualityProfileServerEventsListener = componentModel.GetService<IQualityProfileServerEventsListener>();
+            qualityProfileServerEventsListener.ListenAsync().Forget();
```

Comment:
This instance isn't disposed (unlike the equivalent `IssueServerEventsListener`).
Does it need to be?

Reply:
That's not a helpful reply. Why not?

Reply:
There's nothing to dispose there. It's not responsible for disposing it's dependencies.

Reply:
This class is logically similar to `IssueServerEventsListener`.
The `IssueServerEventsListener` isn't responsible for disposing its dependencies either, but we do dispose it.

Given that the classes are logically similar, why do we dispose one and not the other?

Reply:
It only disposes of it's internal cancellation token. This class doesn't need that.

Reply:
(it's fine for them to be different, but there should be a reason, and we should record it somewhere e.g. a short comment in the code).

Reply:
But isn't it clear from just looking at the class that there's nothing to dispose?